Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -52,13 +52,25 @@ define([
         element.setAttribute('data-bind', 'css: { "cesium-cesiumInspector-visible" : inspectorVisible, "cesium-cesiumInspector-hidden" : !inspectorVisible}');
         container.appendChild(element);
 
+        var panel = document.createElement('div');
+        this._panel = panel;
+        panel.className = 'cesium-cesiumInspector-dropDown';
+        element.appendChild(panel);
+
         var tilesetPanelContents = document.createElement('div');
+        tilesetPanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var displayPanelContents = document.createElement('div');
+        displayPanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var updatePanelContents = document.createElement('div');
+        updatePanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var loggingPanelContents = document.createElement('div');
+        loggingPanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var tileDebugLabelsPanelContents = document.createElement('div');
+        tileDebugLabelsPanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var stylePanelContents = document.createElement('div');
+        stylePanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var optimizationPanelContents = document.createElement('div');
+        optimizationPanelContents.className = 'cesium-cesiumInspector-sectionContent';
```

Comment:
A lot of code duplication. You can prevent it by using `CesiumInspector::addSection` (remove `makeSection` later).